### ISSUE TITLE
[FEATURE] a working-directory option?

### ISSUE THREAD

**Issue by AskAlice** [creyD/prettier_action#42]
**What would you like to change about the program?**

We have a monorepo with multiple npm packages, and run git actions on each one depending on whether or not files were changed in said package.

We have to specify the directory 3 times in the prettier options.
I think this could be simplified by being able to provide a `working-directory` option that chooses where prettier gets CD'd to before running it in CI, then we wouldn't have to specify the config file either.
**Why do you think this is a cool idea?**

It would conslidate the commandline
from:

```yaml
      - name: Frontend Prettier
        uses: creyD/prettier_action@v3.3
        with:
          dry: true
          prettier_options: "--config redacted/redacted/readacted/redacted/app/.prettierrc --ignore-path redacted/redacted/readacted/redacted/app/.prettierignore  --write redacted/redacted/readacted/redacted/app/**/*.{json,md,scss,yaml,yml,js,jsx,tsx,ts}"
```

to:

```yaml
      - name: Frontend Prettier
        uses: creyD/prettier_action@v3.3
        with:
          dry: true
          working-directory: "redacted/redacted/readacted/redacted/app/"
          prettier_options: "--write **/*.{json,md,scss,yaml,yml,js,jsx,tsx,ts}"
```

**Comment by creyD** [comment]
I understand, I think this should be possible.

**Comment by creyD** [comment]
This is implemented with 8efb5a8, would you mind checking if it does what you want to do? @AskAlice

**Comment by creyD** [comment]
This is available in v4.1 now.
